src/frontend/src/components/common/Header/Header.tsx

```diff
@@ -129,9 +129,11 @@ function Header({ user }: Props) {
             alt="Go to the Government of British Columbia website"
           />
         </a>
-        <h1 className="header-h1">
-          Digital Privacy Impact Assessment (DPIA) <span>beta</span>
-        </h1>
+        <p className="header-h1">
+          <strong>
```

Comment:
If possible, I would like to refrain from the usage of `<strong>` as a way of styling text. Think of how we speak when emphasizing a word in a sentence. That is how `<strong>` should be used, IMO. I'm happy to discuss other opinions on this / how I may be incorrect.